longshot/application/main.tf

```diff
@@ -0,0 +1,218 @@
+variable "pipeline" {}
```

Comment:
Hey Dave, is this file a copy of some other .tf in DS codebase with minor modifications?

Reply:
Sort of. I'd previously configured each application instance independently (for example, [northstar](https://github.com/DoSomething/infrastructure/blob/22bbb8a7c0b901dde3269d509589b91ea48525a1/dosomething/northstar/main.tf) and [northstar-dev](https://github.com/DoSomething/infrastructure/blob/master/dosomething-dev/northstar/main.tf)). This is a first stab at standardizing this configuration into reusable modules that we can share amongst environments or similar apps.

Reply:
General suggestion, not necessary to address in this PR: it is good to branch/clone whatever the existing file so that we get a diff in git. I think this reduces cognitive burden on the reader and in the absence of templates .tf files allows to see which sections are shared among apps. I would imagine one possible option: northstar being copy of northstar-dev.

Reply:
And another nit, once you do create a common template, it might make sense to add comments around parts that are not self-explanatory.